custom/icds_reports/templates/icds_reports/icds_app/mobile/location_filter.html

```diff
@@ -0,0 +1,22 @@
+{% load staticfiles %}
```

Comment:
ported from https://github.com/dimagi/mobile-dashboard-prototype/pull/13/files?w=1#diff-d5cf38ad34530367ab0e17b5cfcfdee2

Reply:
though the actual angular logic gets changed a lot later and can probably be ignored in this commit.